packages/server/graphql/private/mutations/connectSocket.ts

```diff
@@ -85,7 +85,8 @@ const connectSocket: MutationResolvers['connectSocket'] = async (
     userId,
     traits: {
       isActive: true,
-      featureFlags: user.featureFlags
+      featureFlags: user.featureFlags,
+      highestTier: user.tier
```

Comment:
Should we change this to `tier` instead of `highestTier` in Segment?

Reply:
Unfortunately It was [called `highestTier`](https://github.com/ParabolInc/parabol/blob/master/packages/server/utils/setUserTierForUserIds.ts#L70) and populated that way already today. Renaming this would cause two separate properties in Amplitude which is more confusing I think... I don't have much tension here.